diff --git a/panopoly_admin.make b/panopoly_admin.make
index b864914..0108848 100644
--- a/panopoly_admin.make
+++ b/panopoly_admin.make
@@ -25,6 +25,7 @@ projects[admin_views][subdir] = contrib
 
 projects[save_draft][version] = 1.4
 projects[save_draft][subdir] = contrib
+projects[save_draft][patch][1409000] = https://www.drupal.org/files/save_draft-per_content_type-1409000-12_2.patch
 
 ; Admin Toolbar Modules
 
